ginivision/src/main/java/net/gini/android/vision/internal/camera/api/CameraController.java

```diff
@@ -41,7 +47,8 @@
 
     private boolean mPreviewRunning = false;
     private AtomicReference<CompletableFuture<Boolean>> mFocusingFuture = new AtomicReference<>();
-    private AtomicReference<CompletableFuture<Photo>> mTakingPictureFuture = new AtomicReference<>();
```

Comment:
I really wish the commit wasn't full of those line breaks. It's distracting and pushes attention away from the important changes. I'd prefer introducing line length limits to be a separate commit.